app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java

```diff
@@ -0,0 +1,191 @@
+//        AnimationUtils.loadAnimation(getApplicationContext(),
+//        R.anim.button_animation);
+//        tvSkip = findViewById(R.id.tv_skip);
+
+        final List<ScreenItem> mList = new ArrayList<>();
+        mList.add(new ScreenItem("Bank Account Management",
+                String.valueOf(R.string.management), R.drawable.management));
+        mList.add(new ScreenItem("Tracking Spending Habits",
+                String.valueOf(R.string.tracking), R.drawable.track_money));
+        mList.add(new ScreenItem("Customer Support",
+                String.valueOf(R.string.support), R.drawable.customer_support));
+
+
+        // setup viewpager
+        viewPager = findViewById(R.id.screenViewPager);
+        introViewPagerAdapter = new IntroViewPagerAdapter(this, mList);
+        viewPager.setAdapter(introViewPagerAdapter);
+
+        // setup tablayout with viewpager
+
+        tabIndicator.setupWithViewPager(viewPager);
+
+
+        // next button click Listner
+
+        btnNext.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+
+                position = viewPager.getCurrentItem();
+                if (position < mList.size()) {
+
+                    position++;
+                    viewPager.setCurrentItem(position);
+
+
+                }
+
+                if (position == mList.size() - 1) { // when we rech to the last screen
+
+                    loaddLastScreen();
+                }
+            }
+        });
+
+        // tablayout add change listener
+
+
+        tabIndicator.addOnTabSelectedListener(new TabLayout.BaseOnTabSelectedListener() {
+            @Override
+            public void onTabSelected(TabLayout.Tab tab) {
+
+                if (tab.getPosition() == mList.size() - 1) {
+
+                    loaddLastScreen();
+
+                }
+
+
+            }
+
+            @Override
+            public void onTabUnselected(TabLayout.Tab tab) {
+
+            }
+
+            @Override
+            public void onTabReselected(TabLayout.Tab tab) {
+
+            }
+        });
+
+
+        // Get Started button click listener
+
+        btnGetStarted.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+
+
+                //open main activity
+
+                Intent mainActivity =
+                        new Intent(getApplicationContext(), LoginActivity.class);
+                startActivity(mainActivity);
+
+                savePrefsData();
+                finish();
+
+
+            }
+        });
+
+    }
+
+    private boolean restorePrefData() {
+
+
+        SharedPreferences pref =
+                getApplicationContext().getSharedPreferences("myPrefs", MODE_PRIVATE);
+        Boolean isIntroActivityOpnendBefore =
+                pref.getBoolean("isIntroOpnend", false);
+        return isIntroActivityOpnendBefore;
+
+
+    }
+
+    private void savePrefsData() {
+
```

Comment:
remove this bank line.

Reply:
I want to ask you about this that, I have two option:-
1. The intro splash screen only shows once to the new user and when the second time user enters the app again then it will not show. (even when user not logged in on the first visit)
2. The intro splash screen shows every time to the new user until he logged in.
So, can you please tell me which idea is good?